packages/css-engine/src/core/to-value.ts

```diff
@@ -6,10 +6,12 @@ export type TransformValue = (styleValue: StyleValue) => undefined | StyleValue;
 
 const fallbackTransform: TransformValue = (styleValue) => {
   if (styleValue.type === "fontFamily") {
-    const fonts = SYSTEM_FONTS.get(styleValue.value[0])?.stack ?? [
-      styleValue.value[0],
-      DEFAULT_FONT_FALLBACK,
-    ];
+    const fallback =
+      styleValue.value.length > 1
+        ? // Its a custom stack, we won't add a fallback
+          styleValue.value
+        : [...styleValue.value, DEFAULT_FONT_FALLBACK];
```

Comment:
Honestly not a fan of integrating font stack into normal font family values. Makes it impossible to use actual font with such name.

Would change this logic this way
1. use provided value
2. if value is singular try to expand into stack
3. add default fallback if no generic-family in the end (https://developer.mozilla.org/en-US/docs/Web/CSS/font-family)

Reply:
Basically its either a stack name or the stack itself.

Reply:
Ok so what you are saying is that we should check first the length of the input, if its 1 item, then we check if its a stack name, if its more than one item, we treat it as a stack. If its 1 item but doesn't match a stack name - its a singular font family name without a fallback, so we add a default fallback to it.

Reply:
this is exactly what I am doing there, but the logic is written in a way that's too unobvious, going to rewrite it so that this is clear